package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.1",
   "description": "Cosine similarity between two object vectors",
   "author": "Adrian C Shum <adrian@cshum.com> (https://github.com/cshum)",
-  "dependencies": {},
+  "dependencies": {
+    "benchmark": "^2.1.4"
```

Comment:
shouldn't this be part of devDependencies?

Reply:
Yeah, you're right.